os/fs/procfs/fs_procfs.c

```diff
@@ -153,7 +153,8 @@ static const struct procfs_entry_s g_procfsentries[] = {
 #endif
 
 #if defined(CONFIG_PM) && !defined(CONFIG_FS_PROCFS_EXCLUDE_POWER)
-	{"power/domains**", &power_procfsoperations},
+	{"power**", &power_procfsoperations},
```

Comment:
I think, it is not necessray
Could you check this again?

Reply:
We need it

`power**` to read all the power subdirectories and files
`power\*` to read the power directory